bluesky/commandline/zmq_proxy.py

```diff
@@ -58,7 +58,7 @@ def main():
         threading.Thread(target=start_dispatcher,
                          args=('localhost', out_port, args.logfile),
                          daemon=True).start()  # Set daemon to all ipython exit
-                                               # kill all threads
+        #                                        kill all threads
```

Comment:
This does not look nice. I'd recommend to place the comment in one line after the `.start()` or move the comment above that line.

Reply:
Done

Reply:
Thanks.